src/Arcus.Templates.WebApi/Arcus.Templates.WebApi.csproj

```diff
@@ -51,6 +51,14 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Remove="ExampleProviders\HealthReportExampleProvider.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="ExampleProviders\HealthReportExampleProvider.cs" Condition="'$(ExcludeOpenApi)' == 'false'" />
+  </ItemGroup>
+
```

Comment:
As I said with your question: should be removed here.

Reply:
Once you remove these things, the build compilation will succeed, too.

Reply:
Done, but, how can I test this locally ?
I can't compile since appsettings.private.json doesn't exist, but I have no idea what this file should contain.

Reply:
It contains the private secrets and connection strings to test worker and other stuff. We can provide you with one, but I think for the stuff you're testing, you don't need any secrets, so an empty file will do just fine.
Let me know if you experience any problems.

Also, thanks for taking the time with this. We know that this repo isn't an easy one to contribute too bc of the magic that happens with .NET project templates.

Reply:
We have a blog post that explains this process more thoroughly. Maybe that will help you understand better what's happening.
https://www.codit.eu/blog/arcus-net-project-templates/